src/views/Home/components/Milestones/milestones.scss

```diff
@@ -1,3 +1,81 @@
-/* stylelint-disable-line no-empty-source */
+@use "~stylesheets/typography";
+@use "~stylesheets/responsiveness";
 
-/* Delete these lines when you begin development */
+.Milestones {
+  overflow: auto;
+  margin: 0;
+  min-height: 375px;
+  background: url("~images/components/milestones/background.png") repeat-y, var(--dark-purple);
+  background-size: 100% auto;
+
+  &__title {
+    @include typography.anaglyph-title-text;
+
+    margin: var(--extra-extra-large-spacer) 0 var(--medium-spacer) 0;
+    text-align: center;
+  }
+
+  &__container {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: center;
+    max-width: 1500px;
+    margin: var(--medium-spacer) auto;
+  }
+
+  &__box {
+    margin: var(--medium-spacer);
+    min-width: 200px;
+    text-align: center;
+    z-index: 99;
+  }
+
+  &__boxTop {
+    @include typography.anaglyph-title-text;
+
+    margin: 0;
+    padding: 30px 40px;
+    // the frosted glass effect isn't well supported in CSS so the color is hard-coded to best match the design.
+    background: var(--purple-frosted-glass);
+    border-radius: 35px 35px 0 0;
+
+    @include responsiveness.respond(tab-port) {
+      padding: 20px 30px;
+    }
+  }
+
+  &__boxBottom {
+    @include typography.description;
+
+    margin: 0;
+    padding: 10px 20px;
+    background: linear-gradient(180deg, var(--light-purple) 0%, var(--light-purple__gradient) 100%);
+    box-shadow: 0 0 35px -3px var(--heliotrope);
+    text-shadow: 1px 1px rgba(0, 0, 0, 0.15);
+    border-radius: 0 0 35px 35px;
+  }
+
+  &__pencil {
```

Comment:
I suggest adding a container for all these accent graphics with a `max-width` that is appropriate alongside the milestone box `container`.  here's what it looks like at 3MP:
![Screenshot from 2020-12-04 15-18-16](https://user-images.githubusercontent.com/19658882/101224609-f8cfaa80-3643-11eb-8a28-f25cd24af8df.png)

Reply:
Good catch! I wrapped everything inside a container with a `max-width` instead, so we can position the graphics relative to the component. (Otherwise a container for the graphics would just have height 0 since all elements are absolutely positioned, and `bottom` wouldn't work). 

This should also be consistent with the `HeroView`. 

Let me know if that works.